sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamReader.scala

```diff
@@ -249,6 +249,8 @@ final class DataStreamReader private[sql](sparkSession: SparkSession) extends Lo
    * `com.databricks.spark.csv`.</li>
    * <li>`escape` (default `\`): sets the single character used for escaping quotes inside
    * an already quoted value.</li>
+   * <li>`escapeQuoteEscaping` (default `\0`): sets the single character used for escaping
```

Comment:
`the single` -> `a single`. Could you update all of them for csv?

Reply:
OK. I'll do it.